dependencies.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-    gosdkVersion = "0.3.1617"
+    gosdkVersion = "0.3.1694-staging"
```

Comment:
I don't think you should use the staging release in the public repository

Reply:
Good catch! The fix for this issue hasn't made it into a public release yet.